site/content/blog/2025-10-01-zap-updates-september-2025/index.md

```diff
@@ -0,0 +1,446 @@
+authors:
+- simon
+---
+
+## Highlights
+
+As we head into October, here’s a roundup of what’s been happening around ZAP over the past month: new features, enhancements, and behind-the-scenes improvements you should know about.
+
+### Configuring Scan Policies with Alert Tags
+
+You can now [configure scan policies with alert tags](/blog/2025-09-03-configuring-scan-policies-with-alert-tags/) in [Automation Framework](/docs/automate/automation-framework/) plans.
+
+This will make configuring scan policies much more intuitive and maintainable.
+
+### WAVSEP Adoption
+
+ZAP is officially [adopting and maintaining WAVSEP](/blog/2025-09-08-zap-is-adopting-wavsep/) (the Web Application Vulnerability Scanner Evaluation Project). 
+
+WAVSEP provides a testbed of web app vulnerabilities used to benchmark and evaluate scanners.
+
+By bringing it in under the ZAP umbrella, the team hopes to keep it updated, extend it to support new types of vulnerabilities, and ensure it's a reliable baseline for evaluation. 
+
+We also welcome contributions from the community!
+
+### Alert De-Duplication
+
+The team is introducing a new mechanism to [suppress duplicate alerts](/2025-09-30-alert-de-duplication/), especially from passive scans. 
+
+* Duplicate alerts often happen when passive scans flag essentially the same issue across multiple URLs (even with trivial differences).
+* ZAP is evolving its Alert Tree logic to adopt naming rules from the Sites Tree, so repeated alerts collapse into a single representative instance. 
+* Systemic alerts (e.g. missing security headers across many endpoints) will get a new SYSTEMIC tag; beyond a certain threshold, additional instances of that alert will be suppressed. 
+* Reporting, APIs, and alerts will include a nodeName field to help with consistent comparisons across scans. 
+
+These changes should help reduce noise, make scan results more meaningful, and improve cross-scan comparisons.
+
+### ZAP Add-on Publishing Guide
+
+We have a new guide for [publishing ZAP Add-ons](/docs/guides/how-to-publish-a-zap-addon/),
+designed to assist developers who want to contribute add-ons to the growing ZAP ecosystem.
+
+The new streamlined process should make it much easier to publish add-ons on the 
+[ZAP Marketplace](/addons/).
+You can now maintain add-ons in your own repos and we'll fork/update them when
+you would like the add-ons to be released.
+
+If you have any questions about developing or publishing add-ons then just ask them on the
+[ZAP Developer Group](https://groups.google.com/group/zaproxy-develop).
+
+## Ongoing Work
+
+We have also been making more improvements to authentication handling.
+
+Beyond the [Microsoft login enhancements](/blog/2025-09-02-zap-updates-august-2025/#microsoft-online-login-support) announced last month, the ZAP team continues to invest in broader authentication support.
+
+The aim is to keep improving ZAP's support for the wide range of authentications flows used in real world apps.
```

Comment:
Sorry I know I bungled my suggestion last time:
"...range of authentication flows used..."

Reply:
singular authentication plural flows

Reply:
ok now?